pubs/commands/import_cmd.py

```diff
@@ -49,11 +49,17 @@ def many_from_path(bibpath):
 
     biblist = []
     for filepath in all_files:
-        biblist.append(coder.decode_bibdata(read_text_file(filepath)))
+        try:
+            biblist.append(coder.decode_bibdata(read_text_file(filepath)))
+        except coder.BibDecodingError:
+            ui.error("Could not parse bibtex at {}. Aborting import.".format(filepath))
+            ui.exit()
```

Comment:
Do we want to abort here? I often found that when there is a problem, I want as much data about it as possible. Even if we decide here to abort the import, we could parse the other files and provide feedback for them too. Are all files unreadable? Just one? Just a bunch? Imagine there are 10 files that have the same problem. I would like to know their name right away, rather than fixing one, doing the import again, and getting a new name, 10 times.

Reply:
True but my point was that if there is an issue, the user might want to fix the files before importing. In particular it might be an issue to have to re-run an import on the same bunch of partly-imported files because of having to force the overwrite option on the second run.

Since it seems that both are valid use cases, I will add an option for ignoring read errors (whether the file is not found or the bibtex is malformed) and mention it in error messages. That way it will also make the behavior with wrongly formatted files coherent with entries that do not pass the check.